### ISSUE TITLE
Test failure with up-to-date stack

### ISSUE THREAD

**Issue by ArchangeGabriel** [Unidata/netcdf4-python#801]
```
......................../build/python-netcdf4/src/netCDF4-1.4.0/test/tst_types.py:92: UserWarning: WARNING: missing_value not used since it
cannot be safely cast to variable data type
  assert_array_equal(v3[:],-1*NP.ones(n2dim,v3.dtype))
..........................................................................E.
======================================================================
ERROR: runTest (tst_cdf5.test_cdf5)
testing NETCDF3_64BIT_DATA format (CDF-5)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/python-netcdf4/src/netCDF4-1.4.0/test/tst_cdf5.py", line 19, in setUp
    v = nc.createVariable('var',np.uint8,'dim')
  File "netCDF4/_netCDF4.pyx", line 2533, in netCDF4._netCDF4.Dataset.createVariable
  File "netCDF4/_netCDF4.pyx", line 3488, in netCDF4._netCDF4.Variable.__init__
  File "netCDF4/_netCDF4.pyx", line 1733, in netCDF4._netCDF4._ensure_nc_success
RuntimeError: NetCDF: One or more variable sizes violate format constraints

----------------------------------------------------------------------
Ran 100 tests in 17.025s

FAILED (errors=1)
not running tst_unicode.py ...
not running tst_dap.py ...

netcdf4-python version: 1.4.0
HDF5 lib version:       1.10.2
netcdf lib version:     4.6.1
numpy version           1.14.3
cython version          0.28.2
foo_bar
```

Please tell me if you need more information.

**Comment by jswhit** [comment]
Fix in pull request #802

**Comment by jswhit** [comment]
pull request #802 merged.  Those warnings are harmless.
